editors/emacs/Portfile

```diff
@@ -274,7 +272,13 @@ variant nativecomp description {Builds emacs with native compilation support} {
     compiler.cpath-prepend         ${prefix}/include/gcc${gcc_v}
     compiler.library_path-prepend  ${prefix}/lib/gcc${gcc_v}
     configure.ldflags-append       "-Wl,-rpath ${prefix}/lib/gcc${gcc_v}"
-    build.args-append              NATIVE_FULL_AOT=1
+
+    if {$subport eq "emacs-devel" || $subport eq "emacs-app-devel"} {
+        configure.args-replace     --with-native-compilation \
+                                   --with-native-compilation=aot
+    } else {
+        build.args-append          NATIVE_FULL_AOT=1
+    }
```

Comment:
On first launch I'm seeing a lot of internal packages being natively compiled. Are we sure that `NATIVE_FULL_AOT` is no longer needed?

Reply:
I don't fully understand this, but it looks like `--with-native-compilation=aot` and `NATIVE_FULL_AOT=1` do the opposite thing (the former appears wrong)?

https://github.com/emacs-mirror/emacs/blob/80c3fadfd57ee797404c80b8e88371c53edecfe5/lisp/Makefile.in#L35-L42

Reply:
`NATIVE_SKIP_NONDUMP` is used [like this](https://github.com/emacs-mirror/emacs/blob/68e2a9cd92996dd55ff9599dc1bf99c70070cddb/lisp/Makefile.in#L377) so actually I guess the flags are OK.